vantage6/vantage6/cli/test/client_script.py

```diff
@@ -85,9 +85,16 @@ def cli_test_client_script(
     """
     if script is None and task_arguments is None:
```

Comment:
I like this syntax more
```suggestion
    if not (script or task_arguments):
```